packages/plugins/plugin-deck/src/components/DeckLayout/DeckLayout.tsx

```diff
@@ -33,27 +34,28 @@ import { useLayout } from '../LayoutContext';
 
 export type DeckLayoutProps = {
   showHintsFooter: boolean;
-  overscroll: Overscroll;
-  flatDeck?: boolean;
   toasts: ToastSchema[];
-  onDismissToast: (id: string) => void;
   // TODO(burdon): Rename planks or just items?
   layoutParts: LayoutParts;
   attention: Attention;
+  // TODO(burdon): Remove.
+  flatDeck?: boolean;
```

Comment:
This is a setting just like wallpaper… why remove?

Reply:
old TODO (i removed the thing!)